packages/cxl-ui/src/components/cxl-credential.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import QRCode from 'qrcode';
+import '@conversionxl/cxl-lumo-styles';
+import cxlCredentialStyles from '../styles/cxl-credential-css.js';
+
+@customElement('cxl-credential')
+export class CxlCredential extends LitElement {
+  static get styles() {
+    return [cxlCredentialStyles];
+  }
+
+  @property({ type: Number, attribute: 'credential-id' }) credentialId = 0;
+
+  @property({ type: String, attribute: 'full-name' }) fullName = '';
+
+  @property({ type: String }) title = '';
+
+  @property({ type: String }) type = '';
+
+  @property({ type: String }) date = '';
+
+  @property({ type: String, attribute: 'credential-url' }) credentialUrl = '';
+
+  /**
+   * @type {string}
+   * @default 'light'
+   * @description Theme
+   * - light
+   * - dark
+   */
+  @property({ type: String, attribute: 'theme', reflect: true }) theme = 'light';
+
+  @query('#qr-code') qrCodeCanvas = null;
+
+  static get defaultUrl() {
+    return 'https://cxl.com/certificate';
```

Comment:
Example URL: https://cxl.com/institute/credential/8105420/

```suggestion
    return 'https://cxl.com/institute/credential';
```